packages/modal/src/components/modal/Modal.module.css

```
@@ -6,6 +6,7 @@
   --swui-modal-width: 960px;
   --swui-modal-header-border-color: var(--lhds-color-ui-300);
   --swui-modal-shadow: var(--swui-shadow-modal);
+  --swui-sticky-modal-footer-shadow: var(--swui-shadow-modal);
```

We use consistent prefix for incoming CSS properties in component styling.
These are the properties that the user can override if they want to have custom styling.

`webui-<feature/component-name>-`

So all incoming CSS properties in this file starts with `--webui-modal-`

```suggestion
  --swui-modal-footer-shadow: var(--swui-shadow-modal);
```

---

Fixed!